src/tidecli/utils/file_handler.py

```diff
@@ -218,10 +243,21 @@ def write_metadata(folder_path: Path, metadata: TaskData) -> None:
     """
     metadata_path = Path(folder_path) / METADATA_NAME
     write_mode = "w"
+    course_metadata: TideCourseData = TideCourseData()
+    weekname = metadata.path
+    taskname = metadata.ide_task_id
     if metadata_path.exists():
-        write_mode = "w"
+        try:
+            with open(metadata_path, "r", encoding="utf-8") as file:
+                smetadata = json.load(file)
```

Comment:
naming typo

Reply:
not typo, it mean string metadata, mayby change to metadata_str?

Reply:
metadata_str is more clear, yes. Doesn't json.load return a dict (or a list) there and not a string?

Reply:
it is json, name changed